auditlog_tests/test_settings.py

```diff
@@ -29,7 +29,9 @@
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.postgresql",
-        "NAME": os.getenv("TEST_DB_NAME", "auditlog_tests_db"),
+        "NAME": os.getenv(
+            "TEST_DB_NAME", "auditlog" + os.environ.get("TOX_PARALLEL_ENV", "")
```

Comment:
Why do we need to change the database name?
it was `auditlog_tests_db` before if we don't provide the `TEST_DB_NAME`. and we already defined valued for `TEST_DB_NAME` for [Github ci](https://github.com/jazzband/django-auditlog/blob/4f4699bf6b48a02ee288f0ba28d63b8d7e851bac/.github/workflows/test.yml#L63) and [tox](https://github.com/jazzband/django-auditlog/blob/4f4699bf6b48a02ee288f0ba28d63b8d7e851bac/tox.ini#L25)

Reply:
When tox runs tests in parallel, it passes `TOX_PARALLEL_ENV` environment variable to every parallel worker. We use this value to make sure that every parallel worker uses its own database and can therefore drop or recreate it and run the migrations on it without causing race conditions with other workers.